### DISCUSSION TITLE
2024 3Q <성장 수집가> 김찬란의 OKR

### DISCUSSION THREAD

**Discussion by seriousran** [Pseudo-Lab/growth-collector#8] [우리들의 성장 목표!]
O: DevRel이 되었지만, Dev 감각 잃지 않기!
- KR1: markdown 기반 인명사전 뷰어 만들기
- KR2: copilot 등을 활용하여 OKR 페이지 개발
- KR3: make를 활용한 자동화 서비스 개발

**Comment by seriousran** [comment]
Action Item
1. markdown viewer & editor 조사
2. 이미지 생성 AI 연구개발자 10인 조사!

**Reply by yunhee1** [reply]
파이팅입니다!!

**Reply by seriousran** [reply]
- Flutter 환경 구축 + Cursor IDE 실험 => 블로깅해야하지 않았나... (반성)
- 10인 조사를 안했다... (반성)
- 퇴근 후 노트북을 너무 펼치지 못했다... (반성)

**Comment by seriousran** [comment]
Action Item
- flutter기반의 markdown viewer & editor 조사
- 이미지 생성 AI 연구개발자 신규 5인 조사!
- 이미지 생성 AI 연구개발자 1인 CV(?) 조사!

**Reply by seriousran** [reply]
- 요런거를 열심히 찾기 보다는, parsing을 해서 쓰는게 낫겠다. flutter 사용할만한 template은 찾음
- ControlNet 논문 1저자 Lvmin Zhang(Stanford Univ.), PIXART-α: 논문 1저자 [Junsong Chen](https://lawrence-cj.github.io/), Black Forest Labs의 인물들(Andreas Blattmann, Axel Sauer, Dominik Lorenz, Dustin Podell, Frederic Boesel, Harry Saini, Jonas Müller, Kyle Lacey, Patrick Esser, Robin Rombach, Sumith Kulal, Tim Dockhorn, Yam Levi, Zion English)
- CV 조사는 하지 못함..ㅠㅠ

**Comment by seriousran** [comment]
Action Item
1. 마법학교
  - 모임 SNS 포스팅
  - 초대 시스템으로의 전환 실험
  - 기존 규칙 정리
2. DevRel 5인 조사
3. 공동 실험에 대한 아이디어
  - 남들이 비웃을만한 것?
  - 나중이 아니라 바로 할 수 있는 것 (=준비가 많이 필요하지 않은 것)

*작성이 늦었습니다...

**Comment by seriousran** [comment]
Action Item (11/1-14)
1. SNS post 열심히 하기
  - 수많은 행사들!
  - 마법학교의 모임 후기
  - 컨텐츠: 기술적 레버리징
2. engagement
  - 노션 기반 CRM 자동화?!

**Reply by yunhee1** [reply]
2주간 파이팅입니다-!

**Reply by SunandBean** [reply]
파이팅입니다!!

**Reply by soncy0311** [reply]
남은 4일도 화이팅 입니다!

**Reply by seriousran** [reply]
미진한 점
- 기술적 레버리징 컨텐츠 제작
- Notion CRM 자동화
